lib/getConfigSemantic.js

```diff
@@ -14,6 +16,11 @@ const { Signale } = require("signale");
  */
 async function getConfigSemantic({ cwd, env, stdout, stderr, logger }, options) {
 	try {
+		// Lazy-load semantic-release ESM module
+		if (!semanticGetConfig) {
+			semanticGetConfig = (await import("semantic-release/lib/get-config.js")).default;
+		}
```

Comment:
Can we use concurrent `import` and `require` flows with try-catches to keep legacy support here?